hack/quicklab/templates/test-pvc.yaml

```diff
@@ -1,7 +1,7 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: test-claim
```

Comment:
why is this change part of the pr?

Reply:
Actually it is not related to this pr. Let me rollback.